src/Interval.h

```diff
@@ -14,7 +14,13 @@ namespace Internal {
 struct Interval {
 
     /** Exprs to represent positive and negative infinity */
-    HALIDE_EXPORT static Expr pos_inf, neg_inf;
+    static Expr pos_inf, neg_inf;
```

Comment:
Why not just make getter methods ```pos_inf()``` and ```neg_inf()``` and move the constants into an anonymous namespace in the file?

Reply:
Or make them static in the function itself.

Reply:
I did it this way to avoid code churn -- if there's no reluctance to that approach I'll do that instead.

Reply:
We have lots of semi-bounded intervals. Within the compiler it could be a performance hit to need to call a function to construct one.

Reply:
In that case I can make the actual `Expr` values private, which will allow for efficient construction.

Reply:
True, the main requirement is for has_upper_bound / has_lower_bound to be fast. Explicit use of pos_inf / neg_inf elsewhere in the compiler could maybe be replaced with helpers to construct intervals that are unbounded above/below